ql/src/java/org/apache/hadoop/hive/ql/parse/ReplicationSpec.java

```diff
@@ -404,20 +402,6 @@ public SCOPE getScope(){
     }
   }
 
-  public boolean isMigratingToTxnTable() {
-    return isMigratingToTxnTable;
-  }
-  public void setMigratingToTxnTable() {
-    isMigratingToTxnTable = true;
-  }
-
-  public boolean isMigratingToExternalTable() {
```

Comment:
Is migration to even external table also being removed?

Reply:
yes